root/static/scripts/timeline.js

```diff
@@ -128,7 +128,8 @@ class TimelineViewModel {
                     var rateBounds = line.calculateRateBounds(
                         line.rateData().data,
                         line.rateData().thresholds,
-                        {min: self.zoom.xaxis.min(), max: self.zoom.xaxis.max()});
+                        {min: self.zoom.xaxis.min(), max: self.zoom.xaxis.max()}
```

Comment:
Here should i make the object a little more clear? 
```suggestion
                        {
                            min: self.zoom.xaxis.min(), 
                            max: self.zoom.xaxis.max()
                          }
```